_benchmark/user-guide/running-workloads.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: default
+title: Running a workload
+nav_order: 9
+parent: User guide
+---
+
+# Running a workload
+
+Once you understand all of the individual components of a OpenSearch Benchmark (OSB) [workload](add-link-to-understanding-workloads), its time to run your first workload. 
+
+## Step 1: Find the workload name
+
+To find the workload name, use the following command:
+
+```
+opensearch-benchmark list workloads 
+```
+{% include copy.html %}
+
+A list of all workloads supported by OpenSearch Benchmark appears. Review the list and select the workload that's most similar to the [use case] of your cluster.
+
+## Step 2: Running the test
+
+After you've selected the workload, you can invoke the workload using the `opensearch-benchmark execute-test` command. Replace  `--target-host` with the `host:port` pairs for your cluster and `--client-options` with any authorization options required to access the cluster. The following example runs the `nyc_taxis` workload on a localhost for testing purposes. If you want to run a test on your own cluster, see [Running the workload on your own cluster](#running-a-workload-on-your-own-cluster).
+
+```bash
+opensearch-benchmark execute-test --pipeline=benchmark-only --workload=nyc_taxis --target-host=https://localhost:9200 --client-options=basic_auth_user:admin,basic_auth_password:admin,verify_certs:false
+```
+{% include copy.html %}
+
+
+Results from the test appear in the directory set by `--output-path` option in `workloads.json`.
+
+### Test mode
+
+If you want run the test in test mode to make sure your workload operates as intended, add the `--test-mode` option to the `execute-test` command. Test mode ingests only the first 1000 documents from each index provided and runs query operations against them.
+
+To use test mode, create a `<index>-documents-1k.json` file that contains the first 1000 documents from `<index>-documents.json` using the following command:
+
+```
+head -n 1000 <index>-documents.json > <index>-documents-1k.json
+```
```

Comment:
We don't need this in running a workload section because this is specific to users who are manually creating their own workloads. 

Instead, we can say that all official workloads use files such as `<index>-documents-1k.json` files that contain the first 1000 documents of `<index>-documents.json`